internal/services/page_rule/schema.go

```diff
@@ -114,6 +115,13 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 					"disable_zaraz": schema.BoolAttribute{
 						Optional: true,
 					},
+					"edge_cache_ttl": schema.Int64Attribute{
+						Optional: true,
+						Validators: []validator.Int64{
+							int64validator.AtLeast(7200),
```

Comment:
this seems like alot for edge cache. is this correct?

Reply:
according to our open api schema this is correct. give me some time to fact check the actual code though.

Reply:
according to the code, edge cache ttl can be as little as 0. I updated this PR. @mgirouard we're going to want to update the schemas sometime this week.